firstvoices/backend/serializers/base_serializers.py

```diff
@@ -7,15 +7,13 @@
 base_id_fields = ("id", "title", "url")
 
 
```

Comment:
Rough diff as usual, sorry. I extracted the site content url field to a mixin so it could be used for models that don't have a title. (Like Person.)